src/actions/preferences.ts

```diff
@@ -25,22 +25,19 @@ export function deletePreferences(userId: string, preferences: PreferenceType[])
         const myPreferences = getMyPreferencesSelector(state);
         const currentPreferences = preferences.map((pref) => myPreferences[getPreferenceKey(pref.category, pref.name)]);
 
-        dispatch({
-            type: PreferenceTypes.DELETED_PREFERENCES,
-            data: preferences,
-            meta: {
-                offline: {
-                    effect: () => Client4.deletePreferences(userId, preferences),
-                    commit: {
-                        type: PreferenceTypes.DELETED_PREFERENCES,
-                    },
-                    rollback: {
-                        type: PreferenceTypes.RECEIVED_PREFERENCES,
-                        data: currentPreferences,
-                    },
-                },
-            },
-        });
+        try {
```

Comment:
I kept these, but I'm not convinced they are /useful/ without first-class support in the WebApp. Notably, the UI was a bit wonky when I reproduced these steps.

Reply:
I feel like I was adding Preferences right around the time we were getting into the idea of having more optimistic actions, so that might be why this is here. I don't think it's the first time I've said this, but I think it might end up being better to be optimistic about specific preference changes (like favouriting a post or setting your theme) while making the general case non-optimistic for simplicity's sake.